web/frontend/src/components/slam.vue

```diff
@@ -56,7 +56,7 @@ const fetchSLAMMap = (name: string): Promise<Uint8Array> => {
       const chunk = res.getPointCloudPcdChunk_asU8();
       chunks.push(chunk);
     });
-    getPointCloudMap.on('status', (status: { code: number, details: string, metadata: string }) => {
+    getPointCloudMap.on('status', (status: { code: number, details: string, metadata: grpc.Metadata }) => {
```

Comment:
okay, now this is just a typing change - no logic changes. see https://github.com/viamrobotics/rdk/pull/2235#discussion_r1170419361 for more context